src/en/flamecomics/src/eu/kanade/tachiyomi/extension/en/flamecomics/FlameComics.kt

```diff
@@ -4,62 +4,166 @@ import android.graphics.Bitmap
 import android.graphics.BitmapFactory
 import android.graphics.Canvas
 import android.graphics.Rect
-import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import android.util.Log
+import eu.kanade.tachiyomi.network.GET
 import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
 import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
 import okhttp3.Interceptor
 import okhttp3.MediaType.Companion.toMediaType
 import okhttp3.Protocol
+import okhttp3.Request
 import okhttp3.Response
 import okhttp3.ResponseBody.Companion.toResponseBody
 import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import uy.kohesive.injekt.injectLazy
 import java.io.ByteArrayOutputStream
+import java.net.URLDecoder
 
-class FlameComics : MangaThemesia(
-    "Flame Comics",
-    "https://flamecomics.xyz",
-    "en",
-    mangaUrlDirectory = "/series",
-) {
-
-    // Flame Scans -> Flame Comics
-    override val id = 6350607071566689772
+class FlameComics : ParsedHttpSource() {
 
     override val client = super.client.newBuilder()
         .rateLimit(2, 7)
         .addInterceptor(::composedImageIntercept)
         .build()
 
-    override val pageSelector = "div#readerarea img:not(noscript img)[class*=wp-image]"
+    // Flame Scans -> Flame Comics
+    override val id = 6350607071566689772
```

Comment:
if old URLs aren't compatible with the new version remove this and bump `versionId` to force the migration